zulip_bots/zulip_bots/bots/github_detail/test_github_detail.py

```diff
@@ -89,3 +90,45 @@ def test_too_many_request(self) -> None:
 
         with self.mock_config_info(self.mock_config):
             self.verify_reply(request, bot_response)
+
+    def test_owner_and_repo_not_specified(self) -> None:
+        request = '/#1'
+        bot_response = 'Failed to detect owner and repository name.'
+        with self.mock_config_info(self.empty_config):
+            self.verify_reply(request, bot_response)
```

Comment:
This test looks like it still needs a mock_http_info.

Reply:
Since, it doesn't has a `owner or repo` it doen't access the `get_details_from_github` and hence doesn't need a mock_http_info.

Reply:
Right, makes sense.